landbosse/excelio/XlsxReader.py

```diff
@@ -145,17 +192,16 @@ def read_xlsx_and_fill_defaults(self, input_xlsx, project):
         incomplete_input_dict['overtime_multiplier'] = project['Overtime multiplier']
         incomplete_input_dict['allow_same_flag'] = True if project['Allow same flag'] == 'y' else False
 
-        # self.default_input_dict['markup_contingency'] = 0.03  # management cost
-        # self.default_input_dict['markup_warranty_management'] = 0.0002
-        # self.default_input_dict['markup_sales_and_use_tax'] = 0  # management cost
-        # self.default_input_dict['markup_overhead'] = 0.05  # management cost
-        # self.default_input_dict['markup_profit_margin'] = 0.05  # management cost
-
-        incomplete_input_dict['markup_contingency'] = project['Markup contingency']
-        incomplete_input_dict['markup_warranty_management'] = project['Markup warranty management']
-        incomplete_input_dict['markup_sales_and_use_tax'] = project['Markup sales and use tax']
-        incomplete_input_dict['markup_overhead'] = project['Markup overhead']
-        incomplete_input_dict['markup_profit_margin'] = project['Markup profit margin']
+        override_total_mgmt_cost_col_name = 'Override total management cost for distributed (0 does not override)'
```

Comment:
Overriding the management cost, while it was originally conceived for distributed wind, can still be available. Thoughts?

Reply:
Done!